packages/devextreme/testing/runner/Views/Main/RunSuite.cshtml

```diff
@@ -241,6 +242,7 @@
                     // Global CSS
                     'generic_light.css': '@Url.Content("~/packages/devextreme/artifacts/css/dx.light.css")',
                     'material_blue_light.css': '@Url.Content("~/packages/devextreme/artifacts/css/dx.material.blue.light.css")',
+                    'gantt.css': '@Url.Content("~/packages/devextreme/artifacts/css/dx-gantt.css")',
```

Comment:
minor, let's move this into `cspMap` object in falsy section of ternary operator